lona/client/dom-renderer.js

```diff
@@ -49,7 +49,7 @@ Lona.LonaDomRenderer = function(lona_context, lona_window) {
 
                     // properties
                     if(property_names.indexOf(key) > -1) {
-                        if(key != 'value') {
+                        if(key != 'value' && node_attributes[key] === '') {
```

Comment:
I think it is more safe
```suggestion
                        if(key != 'value' && node_attributes[key] !== false) {
```

Reply:
@maratori fixed